string video_path
---
bool success
string message